### PR TITLE
fix(web/applications): return empty string when auth is not enabled (…

### PR DESCRIPTION
**Pull request by lutamKC**
…boas-1341)

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/web/src/server/applications/case/project-team/application-project-team.azure-service.js [hunk 1]
```diff
@@ -67,6 +67,14 @@ export const getAllADUsers = async (ADToken) => {
...
 				userPrincipalName: result.userPrincipalName || ''
 			}))
+			// sort by first name
+			// if first name is the same, sort by surname
+			.sort((usr1, usr2) => {
+				if (usr1.givenName !== usr2.givenName) {
+					return usr1.givenName > usr2.givenName ? 1 : -1;
+				}
+				return usr1.surname > usr2.surname ? 1 : -1;
+			})
```
Comment (lutamKC) [inline apps/web/src/server/applications/case/project-team/application-project-team.azure-service.js:77]: details: the ticket require user to be ordered by first name

File: apps/web/src/server/applications/case/project-team/application-project-team.azure-service.js [hunk 2]
```diff
@@ -77,6 +85,11 @@ export const getAllADUsers = async (ADToken) => {
  * @returns {Promise<ProjectTeamMember[]>}
  */
 const getAllCachedUsers = async (session) => {
+	if (config.authDisabled) {
+		// In development only, do not trigger any Azure request
+		return [];
+	}
+
```
Comment (lutamKC) [inline apps/web/src/server/applications/case/project-team/application-project-team.azure-service.js:92]: on local environments, this prevents requests to GRAPH API

File: apps/web/src/server/applications/case/project-team/applications-project-team.service.js [hunk 3]
```diff
@@ -18,9 +18,13 @@ export const searchProjectTeamMembers = async (searchTerm, allAzureUsers, pageNu
 	const searchResults = allAzureUsers.filter((azureUser) => {
 		const { givenName, surname, userPrincipalName: email } = azureUser;
 
-		const recordString = `${givenName || ''} ${surname || ''} ${email || ''}`.toLocaleLowerCase();
-
-		return recordString.includes(searchTerm);
+		// check whether the search matches perfectly the name, the surname or the email
+		// for example, ‘Mil’ will not return ‘Miles’. Users will need to enter ‘Miles’
+		return (
+			`${givenName || ''}`.toLocaleLowerCase() === searchTerm ||
```
Comment (lutamKC) [inline apps/web/src/server/applications/case/project-team/applications-project-team.service.js:24]: reviewing the ticket I noticed this requirement I didn't see the first time.
Now only exact matches will produce results

**Review by HadrianHughesKC — APPROVED** [review]

File: apps/web/src/server/applications/case/project-team/application-project-team.azure-service.js (hunk 2 above)
Comment (francescobrenci-AR) [reply apps/web/src/server/applications/case/project-team/application-project-team.azure-service.js:92]: I'm wondering if we return a set of mock users when `authDisabled` - similarly to what happens with AD groups

**Pull request merged**